Classes/Controller/GetFileController.php

```diff
@@ -269,5 +274,14 @@ private function buildMetsXml($document)
 
         return $metsXml;
     }
+
+    private function isForbidden($action)
+    {
+        $allowed =
+            array_key_exists('allowedActions', $this->settings)
+            && is_array($this->settings['allowedActions'])
+            && in_array($action, $this->settings['allowedActions']);
+        return !allowed;
```

Comment:
The $-sign is missing. This does not work. 
Should be:
<pre>
return !$allowed;
</pre>